### DISCUSSION TITLE
v1.6.0

### DISCUSSION THREAD

**Discussion by deniscerri** [deniscerri/ytdlnis#98] [General]
## What's Changed

- Added Restrict Filenames Preference
- Added Shared File from the History Card Details
- Fixed Command Templates Export/Import crashing on older devices
- Improved Boot Up Times
- Added Custom Command Templates, Shortcuts, Folder picker in Terminal Screen
- Added Sponsor Block settings in the download card
- Added Generic formats for audio tab, so that you can update them if you want
- Fixed Custom Command Template not scrolling properly in the download card
- Added Terminate Button with ability to remove the app from recents menu
- Added Navigation Component
- Added Command Chip filter in the History Fragment
- Implemented ability to Cut Videos. (Multiple Cuts can be done)
- Added ability to cut a video based on its internal chapters as well
- Fixed Terminal text jumping
- Added automatic format updates as soon as you click the format section (if the item/s have no formats)
- Added Ignore Battery Optimizations
- Added Range Picker in playlist bottom sheet
- Added ability to stack search queries in the search view and push them all at once (u can mix links with queries or playlists)
- fixed download logs text not being highlightable
- fixed app not deleting some cached files
- fixed filenames being too short
- fixed sometimes error output not being shown in terminal output or large outputs
- added keyboard focus when adding template in the terminal
- added remux-video for audio
- added copy url button similar to history card in the download card
- fixed padding on cards
- fixed configured download sheet not scrolling
- Added contextual menus for main and history fragments
- You can batch delete / download / share using them, select all, invert selected
- Implemented Cookies feature with export/import functionality
- Added preference to whether download without metered networks
- Added export to clipboard for the terminal output
- Added ability to download subtitles in a file. You can also write the subtitle language based on yt-dlp formatting
- Added ability to choose your preferred search engine instead of just having youtube
- Implemented ability to fetch common formats for all playlist items and choosing it for all of them (with this all formats will be updated for all items aswell)
- Implemented tablet ui with a side navigation bar
- Added ability to swipe right/left on download cards like queued,cancelled,errored,processing to delete or retry/redownload
- Added ability to long press a format card to show its details
- Added theme picker
- Fixed bottom sheets not showing properly on tablets

and more

## New Contributors
* @Qaz-7 made their first contribution in https://github.com/deniscerri/ytdlnis/pull/56
* @Joyogirl made their first contribution in https://github.com/deniscerri/ytdlnis/pull/60

**Full Changelog**: https://github.com/deniscerri/ytdlnis/compare/v1.5.0...v1.6.0

<hr /><em>This discussion was created from the release <a href='https://github.com/deniscerri/ytdlnis/releases/tag/v1.6.0'>v1.6.0</a>.</em>

**Comment by Qaz-7** [comment]
The words Dark and Light have not been added to Weblate

**Reply by deniscerri** [reply]
its added
